### ISSUE TITLE
Ambiguous reference to member 'asyncAfter(deadline:qos:flags:execute:)'

### ISSUE THREAD

**Issue by MaeseppTarvo** [AladinWay/TransitionButton#7]
Using the Xcode 9 and Swift 4, in the TransitionButton class, the following error occurs:

> Ambiguous reference to member 'asyncAfter(deadline:qos:flags:execute:)'

The solution is to remove the `timer in`, so the functions should be something like this:

```
DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
                self.setOriginalState()
                self.shakeAnimation()
            }
```

**Comment by AladinWay** [comment]
Thank you for reporting the issue. I will check it and push a fix tonight if needed.

**Comment by AladinWay** [comment]
fixed in version 0.3.0 👍  thank you for reporting :)
